### ISSUE TITLE
Support visualization of hierarchic state machines

### ISSUE THREAD

**Issue by szokolai-mate** [ELTE-Soft/txtUML#605]
Currently, hierarchical state machines are supported by both language syntaxes in the form of composite states. As far as I know, composite states work as intended, and are correctly exported into the UML2 model.

This, however is not displayed neither in Papyrus nor in JointJS diagrams.

@djnemeth Please verify that this is correct, and provide us with the un-merged old code pertaining to this issue.

**Comment by djnemeth** [comment]
> Composite states work as intended, and are correctly exported into the UML2 model. This, however is not displayed neither in Papyrus nor in JointJS diagrams. Please verify that this is correct.

Correct. The current version of the abstract diagram aligner and the Papyrus/JavaScript visualizer in _master_ handles composite states as simple (flat) ones.

> Please provide us with the un-merged old code pertaining to this issue.

The code is available in branch _[new/papyrus-export](https://github.com/ELTE-Soft/txtUML/compare/new/papyrus-export)_. The related issue is #214.
